features/meta-theme-color.yml.dist

```diff
@@ -0,0 +1,11 @@
+# Generated from: meta-theme-color.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
+  support:
+    chrome_android: "80"
```

Comment:
https://chromestatus.com/feature/5398016231997440 says Chrome 39, and https://caniuse.com/meta-theme-color doesn't have data for Chrome Android. Is 80 the correct version?

Reply:
moving to other PR